_posts/2024-04-14-blog.md

```diff
@@ -0,0 +1,87 @@
+
+It is often the case that most (if not all) sampled individuals will be featured in a Principal Component Analysis (PCA), which is both a very useful visualisation tool for human aDNA data, and the source of many-a [popgen bingo](https://ancientpopgen-bingo.firebaseapp.com/?game=ancientpopgen-bingo).
+How does missing data affect the projected position of ancient individuals on a PCA plot?
+Can you trust the placement of that low coverage individual in the paper you're reviewing, and if so to what degree?
+
+Let's find out!
+
+## Experimental setup
+
+I started with a dataset of present day individuals on the Human Origins array (\~600k SNPs) <sup>[[1](http://dx.doi.org/10.1534/genetics.112.145037),[2](http://dx.doi.org/10.1038/nature13673),[3](https://doi.org/10.1038/s41559-019-0878-2)]</sup>.
+I picked a set of candidate present-day individuals (plus the 'Loschbour' hunter-gatherer) whose positions roughly spanned the area of each PCA plot.
+I then randomly added missing data to each candidate individual's genotypes at a given rate, creating 200 replicates per missingness rate, per individual.
+I then used smartpca [4](http://dx.doi.org/10.1371/journal.pgen.0020190) to run PCA and projected all replicates (`lsqproject: YES`) across all missingness rates for all candidate individuals. The original candidate individuals were not projected, with the exception of 'Loschbour', which is also the only individual with pseudohaploid genotypes in the experiment.
+
+I focused on four PCA setups that are somewhat prevalent in human aDNA literature:
+
+-   Worldwide PCA
+
+-   Eurasian PCA (à la [Jeong2019](https://doi.org/10.1038/s41559-019-0878-2))
+
+-   West Eurasian PCA (à la [Lazaridis et al. 2014](http://dx.doi.org/10.1038/nature13673))
+
+-   African PCA (à la [Skoglund et. al 2017](http://dx.doi.org/10.1016/j.cell.2017.08.049))
+
+## Results
+
+<sup>(You can right click on any of the images below and select "Open Image in a New Tab" to see them in more detail.)</sup>
+
+### Missing 90% of all genotypes (60.000 SNPs)
+
+| :-- | --: |
+![Worldwide](/assets/media/event_images/2024-04-14-blog/Worldwide.m0.9.png)| ![Eurasian](/assets/media/event_images/2024-04-14-blog/Eurasia.m0.9.png)
+![West Eurasian](/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.9.png)|![African](/assets/media/event_images/2024-04-14-blog/Africa.m0.9.png)
+
+### Missing 97% of all genotypes (18.000 SNPs)
+
+| :-- | --: |
+![Worldwide](/assets/media/event_images/2024-04-14-blog/Worldwide.m0.97.png) |![Eurasian](/assets/media/event_images/2024-04-14-blog/Eurasia.m0.97.png)
+![West Eurasian](/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.97.png) |![African](/assets/media/event_images/2024-04-14-blog/Africa.m0.97.png)
+
+### Missing 99% of all genotypes (6.000 SNPs)
+
+| :-- | --: |
+![Worldwide](/assets/media/event_images/2024-04-14-blog/Worldwide.m0.99.png) |![West Eurasian](/assets/media/event_images/2024-04-14-blog/WestEurasia.m0.99.png)
```

Comment:
Generally I find the changing axis scale lengths very jarring and difficult to read.

Also if I'm comparing  the effect of increasing missingness, I want the same PCA-type next to each  other (so worldwide @ 90, 97, 99) all together.

It's currently very difficult to actually compared. It would also make the axis length changes less jarring.

Also missing proper alt_text for each plot (important for accessibility purposes - something I'm being picky with as I've only myself taking more time to do this recently :sweat_smile: _